client/ayon_usd/standalone/usd/pinning/_pinning_file_generation_funcs.py

```diff
@@ -0,0 +1,367 @@
+import logging
+import json
+import os
+import sys
+import re
+from typing import Dict, List, Optional, Set
+from pxr import UsdShade, Ar, Sdf
+from urllib.parse import urlparse
+
+log = logging.getLogger(__name__)
+
+
+def _normalize_path(path):
+    if (sys.platform.startswith('win')
+        and len(path) > 1 
+        and path[1] == ':'  
+        ):
+        
+        path = path[0].lower() + path[1:]
```

Comment:
Isn't this more readable?
```suggestion
    # On windows force drive letter to lowercase
    if sys.platform.startswith('win'):
        drive, tail = os.path.splitdrive(path)
        drive = drive.lower()
        path = f"{drive}{tail}"
```

Reply:
fair point.